src/tabs/pid_tuning.html

```diff
@@ -1377,6 +1377,32 @@
                                     </div>
                                 </td>
                             </tr>
+                            <tr class="newFilter dynamicNotch dynNotchCount">
+                                <td>
+                                    <input type="number" name="dynNotchCount" step="1" min="1" max="5"/>
```

Comment:
According to this new PR https://github.com/betaflight/betaflight/pull/10673 we would need to change `min="1"` to `min="0"`

Reply:
Done @KarateBrot , I have overlocked this new PR.They are finnal values? I think we need to  good think in these values to avoid MSP game in the future.

Reply:
Should be good now :)